specification/mediaservices/resource-manager/readme.md

```diff
@@ -178,6 +195,14 @@ Please also specify the `--go-sdk-folder=<path to the root directory of your azu
 output-folder: $(go-sdk-folder)/services/preview/mediaservices/mgmt/2018-06-01-preview/media
 ```
 
+### Tag: package-2018-07 and go
+
+These settings apply only when `--tag=package-2018-07 --go` is specified on the command line.
+Please also specify the `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-07' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/mediaservices/mgmt/2018-07-01/media

```

Comment:
Did you intend to release this as a preview package?  All of the swaggers are in the stable directory.

Reply:
No, this should be a stable package.  If there is anything I need to do in the md files please let me know.

Reply:
For Go simply remove the `preview/` part from the `output-folder`.  Not sure about the other languages.

Reply:
Fixed.